sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -3,10 +3,10 @@
 
 #pragma once
 
-#include "body_stream.hpp"
+#include "azure/core/http/body_stream.hpp"
+#include "azure/core/internal/contract.hpp"
```

Comment:
Same here/elsehere. Why did we change from `<internal/contract.hpp>` to `"..."`?

Reply:
See above response